### PR TITLE
Clean up the OTIO reader annotation

### PR DESCRIPTION
**Pull request by eloisebrosseau**
### Clean up the OTIO reader annotation

This PR aims to clean up a bit the code related to the OTIO reader for annotations in Live Review.

### Question
I see that some files for other hooks and schemas are in camel case instead of snake case. Does anyone knows if there is a reason? Is everyone ok with me renaming the files inside `otio_reader` to use snake case instead?

### CODE REVIEW

File: src/plugins/rv-packages/otio_reader/annotation_hook.py [hunk 1]
```diff
@@ -11,18 +11,21 @@
...
             else:
-                range = otio.opentime.TimeRange(
+                time_range = otio.opentime.TimeRange(
                     layer.layer_range["start_time"], layer.layer_range["duration"]
                 )
 
-            relative_time = range.end_time_inclusive()
-            frame = relative_time.to_frames()
+            relative_time = time_range.end_time_inclusive()
+            frame = end_time.to_frames()
```
Comment (bernie-laberge) [inline src/plugins/rv-packages/otio_reader/annotation_hook.py:28]: end_time  ?
Shouldn't it be **relative_time** here instead ?
frame = relative_time.to_frames()

Comment (eloisebrosseau) [reply src/plugins/rv-packages/otio_reader/annotation_hook.py]: You're right. Thank you!

**Review by bernie-laberge — APPROVED** [review]
LGTM

Thank you @eloisebrosseau !

File: src/plugins/rv-packages/otio_reader/annotation_hook.py [hunk 2]
```diff
@@ -70,13 +73,11 @@ def hook_function(in_timeline, argument_map=None) -> None:
 
             for point in layer.points:
                 points = commands.getFloatProperty(points_property)
-                if (
+                if not (
                     len(points) > 1
                     and points[-1] == point.y * global_scale.y
```
Comment (rogernelson) [inline src/plugins/rv-packages/otio_reader/annotation_hook.py:78]: wouldn't you need to change the `and` to `or` to maintain the same logic? Or was the old logic wrong?

**Pull request merged**